report_builder/tests.py

```diff
@@ -327,6 +332,7 @@ def test_filter_cached_property(self):
         response = self.client.get(self.generate_url)
         self.assertNotContains(response, 'lol yes')
 
+    @unittest.skip("Custom field not yet implimented")
```

Comment:
Yes - probably going to remove custom field support. That project never took off really and I think a hstore based solution would be better. Leaving it alone for now.